ios/RNInstabug/InstabugReactBridge.h

```diff
@@ -45,14 +107,15 @@
 
 - (void)setExtendedBugReportMode:(IBGExtendedBugReportMode)extendedBugReportMode;
 
-- (void)setReportTypes:(NSArray*)types;
+- (void)setReportTypes:(NSArray *)types;
 
-- (void)showBugReportingWithReportTypeAndOptions:(IBGBugReportingReportType) type: (NSArray*) options;
+- (void)showBugReportingWithReportTypeAndOptions:(IBGBugReportingReportType)type
```

Comment:
Not sure why this doesn't have a label for the second params, but it looks very odd, so let's add it.